content/en/docs/components/pipelines/v2/concepts/pipeline.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Pipeline"
+description = "Conceptual overview of pipelines in Kubeflow Pipelines"
+weight = 10
+                    
++++
+
+## Pipeline
+
+A *pipeline* is a description of a machine learning (ML) workflow, including:
+
+* the metadata of the pipeline, including its name and description 
+* all of the [components](components) in the pipeline 
+* the execution configurations of each of the component
+* [tasks](task) and how they relate to each other in the form of a root [DAG (Directed Acyclic Graph)](dag)
```

Comment:
Calling out components and tasks without further explaining could be confusing to the users.